feyton/phantomBackend
# feat(location)- Add location Tracking

### What does this PR do?
- Add the bus stop functionality
- Add the finish functionality
- Add the connection state property

> Will be  a great demonstration

### Description of Task to be completed

- [x] Live location updates
- [x] Real updates on all subscribers
- [x] Well documented
- [x] Hosted version available

### How should this be manually tested?


```
git clone 
cd phantom-be-codebandits https://github.com/feyton/phantomBackend.git .
git fetch origin
git checkout <your branch>
npm install
//configure the .env according to the .env.sample file
// start the database
npm run dev
// open browser or postman (Check linked public workplace for testing)
```
